src/main/java/pokecube/core/ai/brain/RootTask.java

```diff
@@ -106,12 +106,6 @@ protected void setWalkTo(final Entity mobIn, final double speed, final int dist)
 
     protected void setWalkTo(final WalkTarget target)
     {
-        if (!(target.getTarget() instanceof EntityPosWrapper) && target != null)
-        {
-            final boolean inRange = target.getTarget().getPos().isWithinDistanceOf(this.entity.getPositionVec(), target
-                    .getDistance());
-            if (inRange) return;
-        }
         this.entity.getBrain().setMemory(MemoryModules.WALK_TARGET, target);
```

Comment:
this probably shouldn't be reverted

Reply:
See if my "Revert the revert of the 1.16 branch" fixed it"?

Reply:
No, now it has reverted even more changes from the main branch